backtrader/stores/ccxtstore.py

```diff
@@ -116,10 +117,19 @@ def cancel_order(self, order_id):
     def fetch_trades(self, symbol):
         return self.exchange.fetch_trades(symbol)
 
+
+    @retry
+    def parse_trade(self, trade, market=None):
```

Comment:
Where this method is used?

Reply:
Sorry, this is not currently used. I think I was testing some ideas and forgot to remove it. 

I can recommit with this removed if you like?

Reply:
yes, please update this commit and push again.